lib/selectors.js

```diff
@@ -1,11 +1,20 @@
 import {createSelector} from 'reselect';
+import {findChildSessions} from './utils/term-groups';
+
+const getRootGroups_ = termGroups => Object.keys(termGroups)
+  .map(uid => termGroups[uid])
+  .filter(({parentUid}) => !parentUid);
 
 const getTermGroups = ({termGroups}) => termGroups.termGroups;
-const getRootGroups = createSelector(
+
+export const getRootGroups = createSelector(
   getTermGroups,
-  termGroups => Object.keys(termGroups)
-    .map(uid => termGroups[uid])
-    .filter(({parentUid}) => !parentUid)
+  termGroups => getRootGroups_(termGroups)
 );
 
-export default getRootGroups;
+// For each rootGroup, it sorts its children
+export const getSortedSessionGroups = createSelector(
+  getTermGroups,
+  termGroups => getRootGroups_(termGroups).reduce((result, {uid}) =>
+    Object.assign(result, {[uid]: findChildSessions(termGroups, uid)}), {})
+);
```

Comment:
digit shortcut for each term are not in the store because it is possible to derive them from the global state. It computes children list for each rootGroup (Tab)